ffd_info_exchange/uscis/templates/select-bonus-services.html

```diff
@@ -8,32 +8,34 @@
 
   {% if hed %}<h1 class="headline-text">{{ hed }}</h1>{% endif %}
```

Comment:
...but it's not strictly related to this PR, so I can handle that separately. Hal indeed.